features/google/src/components/activitiesOverTimeStory/activitiesOverTimeStory.jsx

```diff
@@ -0,0 +1,190 @@
+import React, { useState } from "react";
+
+import i18n from "../../i18n";
+
+import "./activitiesOverTimeStory.css";
+import "./datePicker.css";
+import { PolyChart, Tab, Tabs } from "@polypoly-eu/poly-look";
+
+const monthsAbbreviation = i18n.t("common:months.abbreviation").split(" ");
```

Comment:
Since locale strings are a JSON file, we can use any type we want, not only strings. If this shows up repeatedly, maybe we should take that into account.